src/test/groovy/sprouts/Properties_List_Spec.groovy

```diff
@@ -836,4 +836,42 @@ class Properties_List_Spec extends Specification
             change.index() == 2
     }
 
+    def 'Properties created by adding values to a property list can be set to `null` if the list allows null properties.'()
+    {
+        given : 'A nullable "Vars" instance having a few properties.'
+            var vars = Vars.ofNullable(String.class, "a", null, "c")
+        expect : 'All properties in the property list are nullable.'
+            vars.at(0).allowsNull();
+            vars.at(1).allowsNull();
+            vars.at(2).allowsNull();
+        when : 'We add a empty property.'
+            vars.add("x");
+        then : 'The property list should contain the added property.'
+            vars == Vars.ofNullable(String.class, "a", null, "c", "x")
+        and : 'The added property should also be nullable.'
+            vars.at(3).allowsNull()
+        when : 'We can set the new added property to `null`.'
+            vars.at(3).set(null)
+        then : 'The added property should now be empty'
+            vars.at(3).isEmpty()
+            vars == Vars.ofNullable(String.class, "a", null, "c", null)
+    }
+
+    def 'Properties created by adding values to a property list cannot be set to `null` if the list does not allow null properties.'()
+    {
+        given : 'A "Vars" instance having a few properties.'
+            var vars = Vars.of("a", "b", "x", "d")
+        expect : 'All properties in the property list are not nullable.'
+            !vars.at(0).allowsNull();
+            !vars.at(1).allowsNull();
+            !vars.at(2).allowsNull();
+            !vars.at(3).allowsNull();
+        when : 'We add a property.'
+            vars.add("y");
+        then : 'The property list should contain the added property.'
+            vars == Vars.of("a", "b", "x", "d", "y")
+        and : 'The added property should also be not nullable.'
+            !vars.at(4).allowsNull()
+    }
+
```

Comment:
These are nice feature specs! I think we can reintroduce them 
in a new commit at the end...